_includes/ogp.html

```diff
@@ -1,18 +1,22 @@
 {% capture header %}
 
 {% if page.title %}
-<meta property="og:type" content="article" />
+<meta property="og:type" content="website" />
```

Comment:
IMO the "article" is more like a real blog post or a news story, with an author etc. We don't really have that.